### PR TITLE
add new features story

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

Add new stories with platform features
Add hold by click on story (only for desktop currently)

**Screenshots**
image

image
image

image

image

image
image

### CODE REVIEW

File: src/components/Stories/content.js [hunk 1]
```diff
@@ -1,6 +1,234 @@
 import React from 'react'
+import chartImg from './../../assets/stories/platform_features/chart.png'
+import mvrvImg from './../../assets/stories/platform_features/mvrv.png'
+import mmaImg from './../../assets/stories/platform_features/mma.png'
+import distroImg from './../../assets/stories/platform_features/distro.png'
+import renTopsImg from './../../assets/stories/platform_features/ren.png'
+import btcTacImg from './../../assets/stories/platform_features/btc_ta.png'
+import weeklyUpdatesImg from './../../assets/stories/platform_features/weekly.png'
```
Comment (haritonasty) [inline src/components/Stories/content.js]: please, host images [here](https://github.com/santiment/sanbase2/tree/master/assets/static/images/sanbase/Stories)

File: src/components/Stories/content.js [hunk 2]
```diff
@@ -1,6 +1,234 @@
...
+        title: 'Token Holder metrics',
+        description: (
+          <>
+            <div>
+              Jan Smirny, Santiment’s data scientist and researcher, dives deep
+              into our <b>Token Holder metrics</b>{' '}
+              <a
+                target='_blank'
+                rel='noopener noreferrer'
+                target='_blank'
```
Comment (haritonasty) [inline src/components/Stories/content.js]: duplicate attr

File: src/components/Stories/content.js [hunk 3]
```diff
@@ -1,6 +1,234 @@
...
+              newly-developed metrics: <b>Maximal Mean Age</b>, and how it
+              enables a radically new way to look at investor behavior.
+            </div>
+
+            <br />
+
+            <i>
+              ‘Every price top so far was accompanied by a drop in the mean coin
+              age and with a significant drop of mean dollar age. ‘
+            </i>
```
Comment (haritonasty) [inline src/components/Stories/content.js]: this text already on image - skip it

File: src/components/Stories/content.js [hunk 4]
```diff
@@ -1,6 +1,234 @@
...
+              coins across the board - and even beat holding ETH:
+            </div>
+
+            <br />
+
+            <i>
+              ‘While more volatile, our distributed portfolio beats the ETH
+              benchmark at almost all points in time.‘
+            </i>
+          </>
```
Comment (haritonasty) [inline src/components/Stories/content.js]: also on the image

File: src/components/Stories/content.js [hunk 5]
```diff
@@ -1,6 +1,234 @@
...
+        )
+      },
+      {
+        image: renTopsImg,
+        title: 'On-chain and social indicators',
+        description: (
```
Comment (haritonasty) [inline src/components/Stories/content.js]: use `isDarkImage: true` for dark background here

**Comment by GarageInc** [comment]
@Partysun , what can we do with a lot of text? Divide into 2 slides or ask Nemo about simplify?

File: src/components/Stories/content.js (hunk 1 above)
Comment (GarageInc) [reply src/components/Stories/content.js]: @haritonasty , why does stories depend from images on other app?

Comment (GarageInc) [reply src/components/Stories/content.js]: Wait backend appoves in their repo https://github.com/santiment/sanbase2/pull/1615

Comment (haritonasty) [reply src/components/Stories/content.js]: for loading them by url

File: src/components/Stories/content.js [hunk 6]
```diff
@@ -1,6 +1,215 @@
...
+              into our <b>Token Holder metrics</b>{' '}
+              <a
+                target='_blank'
+                rel='noopener noreferrer'
+                href='https://insights.santiment.net/read/token-distribution-ratio%3A-new-performance-indicator-for-crypto%3F-%5Bsantiment-analysis%5D-314'
+              >
+                in this backtest
+              </a>
+              , concluding that distributed coins tend to outperform centralized
+              coins across the board - and even beat holding ETH:
```
Comment (haritonasty) [inline src/components/Stories/content.js]: delete `:` at the end

File: src/components/Stories/content.js (hunk 1 above)
Comment (GarageInc) [reply src/components/Stories/content.js]: But by import from app we also load them from url...

Comment (GarageInc) [reply src/components/Stories/content.js]: Example from app: 
image

Comment (GarageInc) [reply src/components/Stories/content.js]: And also strange imports from 'stage' application on production app
image

**Comment by GarageInc** [comment]
@Partysun , what can we do with a lot of text on stories?

1. Ask Nemo
2. Divide into 2 slides
3. Leave as is. I added a hold for click on content.

File: src/components/Stories/content.js (hunk 1 above)
Comment (haritonasty) [reply src/components/Stories/content.js]: But they will be in the bundle (maybe not in main, but will) and increase his size. Or I mess smth?

Comment (GarageInc) [reply src/components/Stories/content.js]: Not in a bundle. For example you can see a direct link for logo 
https://app-stage.santiment.net/static/media/logo-sanbase.fcc9cb23.svg
It is imported in code as external file.

Comment (haritonasty) [reply src/components/Stories/content.js]: ok, thanks. But for common illustrations (that will be use in different products) better to use 1 remote link

**Pull request merged**